.github/workflows/docker-build-push.yml

```diff
@@ -1,17 +1,27 @@
 name: docker-build-push
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - gce-deploy # for testing purposes
+
+env:
+  PROJECT_ID: ${{ secrets.GCE_PROJECT }}
+  GCE_INSTANCE: deploy-pudl-vm
+  GCE_INSTANCE_ZONE: us-central1-a
 
 jobs:
   push_to_registry:
     name: Build Docker image and push to Docker Hub
     runs-on: ubuntu-latest
+    outputs:
+      docker_tags: ${{ steps.docker_metadata.outputs.tags }}
```

Comment:
Does this just make the tags available for other jobs?

Reply:
Yes. I split build and deploy into two jobs but they can totally be one if you like.

Reply:
Nah this seems totally fine. I was just curious how it was working.